targets/lenovo-x1/appvms/chromium.nix

```diff
@@ -53,6 +48,11 @@ in {
         "intel-hda"
         "-device"
         "hda-duplex,audiodev=pa1"
+        # Lenovo X1 integrated usb webcam
+        "-device"
+        "qemu-xhci"
+        "-device"
+        "usb-host,hostbus=3,hostport=8"
```

Comment:
Could this inherit these values from `modules/common/hardware/lenovo-x1/definitions/x1-<gen>.nix` ?
The bus and port values per hardware device are not there yet but could be. That way we could continue decoupling the chromium appvm from the hardware though there's some mode work to do with audio devices but I think the future development of audio architecture would allow that.

Reply:
Yes that could be an option. I think it really depends on how we deal with different hardware definitions in the future - will we support different generations and/or HW SKUs as targets? See also the comment below.